R/downloadableReactTable.R

```diff
@@ -125,6 +132,16 @@ downloadableReactTableUI <- function(id,
 #'                       a reactive expression or a function returning a character string (default = 'data_file')
 #' @param download_data_fxns a \strong{named} list of functions providing the data as return values.
 #'                           The names for the list should be the same names that were used when the table UI was created
+#' @param enale_pagination to enable table pagination (default = FALSE)
+#' @param table_height max table height in pixels. Vertical scroll will be shown after that height value
```

Comment:
remove the word "enable" - none of these options need an enable, that is implied.  Pagination is fine, "global_search" is fine and explanatory by itself.  Let's be more succinct and consistent in naming.  Stay consistent - rename "striped" to "row_striping" to be explicit but it doesn't need an enable.  What is the default for row_highlight and striping?  Set defaults.